pkg/gateway/api/api.go

```diff
@@ -979,6 +983,20 @@ func (s *gatewayService) registerEvents(w http.ResponseWriter, req *http.Request
 				}
 				continue
 			}
+			if err := s.evaluationEventStorage.CreateEvaluationEvent(
```

Comment:
Because CloudSQL doesn't support autoscaling and doing this here would increase the server loading, please move this implementation to the event persister service.